File: KMS/KMS-war/src/java/ws/restful/model/GroupRecommendationBasedOnFollowingRsp.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ws.restful.model;

import entity.GroupEntity;
import entity.UserEntity;
import java.util.List;

/**
 *
 * @author Cassie
 */
public class GroupRecommendationBasedOnFollowingRsp {

    private GroupEntity groupToRecommend;

    private List<UserEntity> followingInGroup;

    public GroupRecommendationBasedOnFollowingRsp() {
    }

    public GroupEntity getGroupToRecommend() {
        return groupToRecommend;
    }

    public void setGroupToRecommend(GroupEntity groupToRecommend) {
        this.groupToRecommend = groupToRecommend;
    }

    public List<UserEntity> getFollowingInGroup() {
        return followingInGroup;
    }

    public void setFollowingInGroup(List<UserEntity> followingInGroup) {
        this.followingInGroup = followingInGroup;
    }
}

File: KMSAngular/src/app/models/GroupRecommendationBasedOnFollowingRsp.ts
import { Group } from '../classes/group';
import { User } from '../classes/user';

export class GroupRecommendationBasedOnFollowingRsp {
    groupToRecommend: Group;
    followingInGroup: User[];

    constructor(groupToRecommend?: Group, followingInGroup?: User[]) {
        this.groupToRecommend = groupToRecommend;
        this.followingInGroup = followingInGroup;
    }
}